docs/reference/cli_subcommands/plotbwdf.rst

```diff
@@ -0,0 +1,15 @@
+plot-bwdf
+==========
+
+Plot BWDFs using ICOHPs or ICOOPs or ICOBIs with respect to bond lengths.
+
+.. hint:: Aliases available
+
+   - plotbwdf
+
+.. argparse::
```

Comment:
Is it nobvious how to switch to COOP?

Reply:
Yes, using help shows all options, it works same way as in automatic analysis / icohp-distance plotter.

Users can use `--coops`  or `--cobis` to plot ICOOPs or ICOBIs respecitvely.